source/codegen/metadata/nifpga/config.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '24.3.0',
+    'c_header': 'NiFpga.h',
+    'c_function_prefix': 'NiFpga_',
+    'c_dll_function_prefix': 'NiFpgaDll_',
+    'service_class_prefix': 'NiFpga',
+    'java_package': 'com.ni.grpc.nifpga',
+    'csharp_namespace': 'NationalInstruments.Grpc.NiFpga',
+    'namespace_component': 'nifpga',
+    'init_function': 'Open',
+    'close_function': 'Close',
+    'code_readiness': 'NextRelease',
```

Comment:
Do have a task under your feature to remove this feature toggle when you are near done.

Reply:
will do that.